docs/trust.md

```diff
@@ -0,0 +1,9 @@
+AEA applications operate within different orders of trustlessness.
+
+For example, using the AEA weather skill application without a ledger for transactions means that clients must 100% trust the weather station that any data it sends is sufficient, including no data at all.
+
+A step up, if you run the weather skill application on a ledger system then the client must again trust the weather station to send sufficient data. However, all transactions are recorded so there is some data verifiability.
```

Comment:
Add: Crucially, the weather station does not need to trust the weather client as it can observe the transaction taking place on the public ledger.